docs/documentation/cn/administrator-guide/doris-on-es/doris-on-es.md

```diff
@@ -0,0 +1,200 @@
+# Doris-On-ES使用手册
```

Comment:
put it to  docs/documentation/cn/extending-doris

Reply:
OK, Please review again.